apps/condo/domains/meter/components/TabContent/Meter.tsx

```diff
@@ -46,6 +48,19 @@ const FULL_WIDTH_DATE_RANGE_STYLE: CSSProperties = { width: '100%' }
 const SORTABLE_PROPERTIES = ['verificationDate', 'source']
 const DEFAULT_DATE_RANGE: [Dayjs, Dayjs] = [dayjs(), dayjs().add(2, 'month')]
 
+
+const getInitialArchivedOrActiveMeter = (router: NextRouter, field: 'isShowActiveMeters' | 'isShowArchivedMeters', defaultValue = false) => {
+    if (field in router.query && isString(router.query[field])) {
+        try {
+            return (JSON.parse(router.query[field]))
+        } catch (error) {
+            console.warn(`Failed to parse property value ${field}`, error)
+            return false
+        }
+    }
+    return defaultValue
```

Comment:
For consistency you may return default value here too. Just in case of someone wants to override it via function argument
```suggestion
            return defaultValue
        }
    }
    return defaultValue
```

Reply:
yes, you're right